packages/core/src/libraries/sign-in-experience/index.ts

```diff
@@ -196,7 +224,10 @@ export const createSignInExperienceLibrary = (
     };
 
     return {
-      ...deepmerge(signInExperience, organizationOverride ?? {}),
+      ...deepmerge(
+        deepmerge(signInExperience, appSignInExperience ?? {}),
```

Comment:
not a blocker, should not merge application-level SIE with default SIE for the third-party apps.

Reply:
i'll update it with some tests in a new pull